### DISCUSSION TITLE
Cannot add labels to PRs

### DISCUSSION THREAD

**Discussion by stavrogin** [renovatebot/renovate#8774] [Request Help]
Hi, 
I am on GitHub Enterprise. Everything works fine, but I cannot see the labels applied to my PRs
Here is my renovate.json

```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base"
  ],
  "commitMessagePrefix": "UPGRADE",
  "labels": ["dependencies"]
}

```
Is it correct?

Thank you,

Flavio

**Comment by rarkins** [comment]
Yes, that's correct. Labels are only applied at tone of PR creation though. If in doubt, log at debug level and look for any mentions of "labels"
